File: app/src/main/java/com/tushar/gautam/grenade/AppInfo.java
package com.tushar.gautam.grenade;

import android.graphics.drawable.Drawable;

public class AppInfo {
    String label;
    String packageName;
    Drawable appicon;

    public AppInfo(String label,String packageName, Drawable appicon){
        this.label = label;
        this.packageName = packageName;
        this.appicon = appicon;
    }

    public String getLabel(){
        return label;
    }
    public String getPackageName(){
        return packageName;
    }
    public Drawable getAppicon(){
        return appicon;
    }
}

File: app/src/main/java/com/example/grenade/appInfo.kt
package com.example.grenade

import android.graphics.drawable.Drawable

public   class  appInfo(var applabel:String,var packageName:String,var appicon:Drawable){


    public  fun appInfo(applabel:String, packageName:String,appicon:Drawable) {
        this.applabel = applabel
        this.packageName = packageName
        this.appicon = appicon
    }

    public fun getAppLabel(): String {
        return applabel
    }
    public fun getpackageName(): String {
        return packageName
    }
    public fun getAppIcon(): Drawable {
        return appicon
    }

}

